Cabal/doc/nix-local-build.rst

```diff
@@ -235,10 +235,10 @@ this folder (the most important two are first):
     a local package are unchanged, ``cabal v2-build`` will skip
     invoking ``setup build`` entirely (saving us from a possibly
     expensive call to ``ghc --make``). The full list of source files
-    participating in compilation are determined using
-    ``setup sdist --list-sources`` (thus, if you do not list all your
-    source files in a Cabal file, you may fail to recompile when you
-    edit them.)
+    participating in compilation is determined using
+    ``cabal sdist --list-only``. Thus if you do not list all your
```

Comment:
Not sure about this, but this seems to be v2- equivalent?

Reply:
This is technically correct, and `cabal (v2-)sdist` command is something people can run easily, so is better.